OpenRA.Game/Graphics/Sprite.cs

```diff
@@ -17,7 +17,6 @@ public class Sprite
 		public readonly Rectangle Bounds;
 		public readonly Sheet Sheet;
 		public readonly BlendMode BlendMode;
-		public readonly float Alpha;
```

Comment:
The alpha multiplication is done at the palette level, so it simply doesn't work here.